doc/contributor/workflow.md

```diff
@@ -9,7 +9,7 @@ You will need:
 * [LLVM](../user/installing-llvm.md)
 * [`libssl`](../user/installing-libssl.md)
 
-You can remove the dependencies on LLVM and `libssl` by setting
+You can remove the dependencies on `libssl` by setting
```

Comment:
In what sense does this remove the dependency on `libssl`? Does it work without it?

Reply:
Yes, it will just not build the C exts (openssl/zlib/syslog). But we still build Sulong no matter what.
But of course openssl/zlib/syslog will not work then.